.builders/lock.py

```diff
@@ -109,6 +109,11 @@ def generate_lock_file(
                 if not is_compatible_wheel(target, python_major, interpreter, abi, platform):
                     continue
 
+                if build and 'WID' in build:
+                    # TODO: we're skipping any wheels with WID in them until we merge
```

Comment:
I'm not sure of historical usage at Datadog, but I always find it useful if a TODO is never naked. TODO(<link to jira ticket>) or TODO(<link to doc>) are much more useful to a maintainer because it allows someone else to get the needed context.